scout/server/blueprints/variant/templates/variant/tx_overview.html

```diff
@@ -84,7 +84,13 @@
                     <div>
                       {{ refseq_id }}<span class="text-muted"> ({{ transcript.transcript_id }})</span>
                       {% if refseq_id in gene.common.primary_transcripts %}
-                      <span class="text-muted">(hgnc-primary)</span>
+                        <span class="text-muted">(hgnc-primary)</span>
+                      {% endif %}
+                      {% if transcript.mane_transcript %}
+                        {% set mane_transcript_parts = transcript.mane_transcript.split('.') %}
```

Comment:
How does it look like, the whole transcript.mane_transcript saved to the database?

Reply:
It's simply the refseq ID of the matched "MANE select" transcript. I'm removing the version number with the split() to be able to match it to the refseq variable (which doesn't have the version number).

I'll try to find time to add tests soon! Sorry about the delay...

Reply:
It's OK! 👍